backend/LexBoxApi/GraphQL/OrgMutations.cs

```diff
@@ -30,12 +30,68 @@ public class OrgMutations
             Members =
             [
                 new OrgMember() { Role = OrgRole.Admin, UserId = userId }
-            ]
+            ],
+            Projects = []
         });
         await dbContext.SaveChangesAsync();
         return dbContext.Orgs.Where(o => o.Id == orgId);
     }
 
+    [Error<DbError>]
+    [Error<NotFoundException>]
+    [Error<AlreadyExistsException>]
+    [UseMutationConvention]
+    [UseFirstOrDefault]
+    [UseProjection]
+    public async Task<IQueryable<Organization>> AcquireProject(
+        LexBoxDbContext dbContext,
+        IPermissionService permissionService,
+        Guid orgId,
+        Guid projectId)
+    {
+        var org = await dbContext.Orgs.FindAsync(orgId);
+        NotFoundException.ThrowIfNull(org);
+        permissionService.AssertCanEditOrg(org);
+        var project = await dbContext.Projects.FindAsync(projectId);
+        NotFoundException.ThrowIfNull(project);
+
+        if (await dbContext.OrgProjects.AnyAsync(op => op.OrgId == orgId && op.ProjectId == projectId))
+        {
+            throw new AlreadyExistsException("Organization already owns this project");
```

Comment:
Should I just let this situation pass without error? The desired state is that the org should own the project, so if that's already the case, then we're already in the desired state. To my way of thinking, that means an error should not be thrown. But AddProjectMember throws the "User is already a member of this project" error in a similar case. However, LeaveProject does *not* throw an error if the leaving user was already not a member, so there's inconsistency.

For now, I chose to follow the same inconsistent logic as AddProjectMember and LeaveProject, where the "add" operaton throws but the "remove" one does not. But it's possible that we'll want to be consistent here instead. Thoughts?

Reply:
yeah let's actually not throw an error, I agree the state is already how we want it so there's no problem.

Reply:
Done in commit a6dd4908.